omod/src/main/webapp/resources/htmlforms/HMIS-HIV-003-HivCareArtCard-ClinicalAssessmentPage.xml

```diff
@@ -38,11 +36,153 @@
                     * object, suitable for use in a URL query string or Ajax request.
                     */
                     location.href = window.location.protocol + "//" + window.location.host +
-                    '/' + OPENMRS_CONTEXT_PATH + '/coreapps/clinicianfacing/patient.page?patientId='
-                    + queryParameters['patientId']; // Causes page to reload
+                        '/' + OPENMRS_CONTEXT_PATH + '/coreapps/clinicianfacing/patient.page?patientId='
+                        + queryParameters['patientId']; // Causes page to reload
                 });
-            </includeIf>
-        </ifMode>
+            </script>
+        </includeIf>
+    </ifMode>
+    <!-- jquery loaded for nonessential purposes. -->
+
+    <style>
+        .nav-link-1 {
+            background-color: #015f6b;
+        }
+
+        .nav-pills .nav-link.active, .nav-pills .show > .nav-link {
+            color: #FFFFFF;
+            background-color: #000000;
+            border-radius: 5px;
+        }
+
+        form .lablex2 span {
+            min-width: 50%;
+            float: left;
+            margin: 1px 0px;
+        }
+
+        form .lablex3 span {
+            min-width: 31%;
+            float: left;
+            margin: 1px 0px;
+        }
+
+        form .lablex4 span {
+            min-width: 25%;
+            float: left;
+            margin: 1px 0px;
+        }
+
+        a {
+            color: #f9f9f9;
+            text-decoration: none;
+        }
+
+        i a {
+            color: #f9f9f9;
+            text-decoration: none;
+        }
+
+        .nav-tabs .nav-link {
+            border: 1px solid #02aec3;
+            border-top-left-radius: .25rem;
+            border-top-right-radius: .25rem;
+            color: blue
+        }
+
+        #final-out-come {
+            margin-top: 26px;
+        }
+
+        .customSubmit {
+            font-weight: bold;
+            width: 70%;
+        }
+
+        .submission-section {
+            width: 100%;
+            align-items: center;
+
+        }
+
+        .encounter-details {
+            margin-top: 68px;
+            margin-left: 120px;
+            width: 70%;
+        }
+
+        .submitButton {
+            width: 50%;
+        }
+
+        form select[disabled], .form select[disabled] {
+            background: #EEE;
+            border: #1dacbd 1px solid;
+            color: #060606;
+            font-style: normal;
+        }
+
+        form select, .form select {
+            min-width: 100%;
+            max-width: 100%;
+        }
+
+        form input {
+            min-width: 100%;
+            max-width: 100%;
+        }
+
+        .add-more-link {
+            color: dodgerblue;
+        }
+
+        .remove-more-link {
+            color: darkred;
+        }
+
+
+    </style>
+    <script type="text/javascript">
+
+        if (jQuery) {
```

Comment:
@slubwama combine this JS initialisation with the one below it no need to separate them.

Reply:
done